### PR TITLE
Differentiate dashboard v1 & v2 banners

### PR DESCRIPTION
**Pull request by esbie**
Adds version 2 banner and moves version 1 specific css out of
AdminHandler

screen shot 2016-03-25 at 11 29 37 am

Unresolved:
- [ ] the linkerd logo needs to be re-rendered with larger font
- [ ] we need to decide if we're going to use Helvetica, DINNextLT, or what

### CODE REVIEW

**Comment by siggy** [comment]
On http://localhost:9990/delegator?router=http ...
screen shot 2016-03-25 at 1 11 18 pm

File: linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/LinkerdAdmin.scala [hunk 1]
```diff
@@ -17,9 +17,9 @@ class LinkerdAdmin(app: App, linker: Linker, config: LinkerConfig) extends Admin
   private[this] val dtabs: Future[Map[String, Dtab]] =
     Future.value(
       linker.routers.map { router =>
-        val RoutingFactory.BaseDtab(dtab) = router.params[RoutingFactory.BaseDtab]
-        router.label -> dtab()
-      }.toMap
+      val RoutingFactory.BaseDtab(dtab) = router.params[RoutingFactory.BaseDtab]
```
Comment (rmars) [inline linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/LinkerdAdmin.scala]: mine keeps doing this too.  it is driving me nuts

**Comment by siggy** [comment]
:+1: :star: thanks!

**Comment by rmars** [comment]
:star: :art: :8ball: yay

**Pull request merged**